templ_attr.cif

```diff
@@ -554,21 +554,26 @@ save_
 
 save_aniso_bij
 
-    _definition.update           2013-03-08
+    _definition.update           2023-02-16
     _description.text
 ;
-     These are the standard anisotropic atomic displacement components
-     in angstroms squared which appear in the structure factor term:
+     These are the standard anisotropic atomic displacement components, in
+     angstroms squared, which appear in the structure factor term:
 
          T = exp{-1/4 sum~i~ [ sum~j~ (B^ij^ h~i~ h~j~ a*~i~ a*~j~) ] }
 
      h = the Miller indices
      a* = the reciprocal-space cell lengths
+
      The unique elements of the real symmetric matrix are entered by row.
 
-     The IUCr Commission on Nomenclature recommends against the use
-     of B for reporting atomic displacement parameters. U, being
-     directly proportional to B, is preferred.
+     The IUCr Commission on Nomenclature recommends against the use of B for
+     reporting atomic displacement parameters. U, being directly proportional
+     to B, is preferred [1].
+
+     Note that U^ij^ = \b^ij^/(2 pi^2^ a*~i~ a*~j~) = B^ij^/(8 pi^2^) [1].
+
+     [1] Trueblood, K. N. et al. (1996). Acta Crystallogr. A52(5), 770-781.
```

Comment:
Given that the Unicode branch is now merged in, we might want to update the text here as well to contain proper Unicode symbols.